lib/pages/events_page.dart

```diff
@@ -0,0 +1,226 @@
+import 'package:aaveg_app/controllers/events_page_controller.dart';
+import 'package:aaveg_app/controllers/nav_bar_controller.dart';
+import 'package:aaveg_app/models/event_modal.dart';
+import 'package:aaveg_app/views/widgets/Events/event_card.dart';
+import 'package:aaveg_app/views/widgets/NavBar/navbar_widget.dart';
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class EventsPage extends StatefulWidget {
+  EventsPage({Key? key}) : super(key: key);
+
+  @override
+  State<EventsPage> createState() => _SampleState();
+}
+
+final NavBarController navBarController = Get.find<NavBarController>();
+GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+class _SampleState extends State<EventsPage> with TickerProviderStateMixin {
+  // finds your controller and uses it
+  final sampleController = Get.find<EventsPageController>();
+  late TabController _tabController;
+  List<Event> eventlist = [
+    Event(
+        name: 'Drama',
+        clusterName: 'Vaayu',
+        type: 'cultural',
+        desc: 'Drama for first years',
+        cupName: 'dramacup',
+        r1Date: '15 june',
+        r2Date: '18 june'),
+    Event(
+        name: 'Drama',
+        clusterName: 'Vaayu',
+        type: 'cultural',
+        desc: 'Drama for first years',
+        cupName: 'dramacup',
+        r1Date: '15 june',
+        r2Date: '18 june'),
+    Event(
+        name: 'Drama',
+        clusterName: 'Vaayu',
+        type: 'cultural',
+        desc: 'Drama for first years',
+        cupName: 'dramacup',
+        r1Date: '15 june',
+        r2Date: '18 june'),
+    Event(
+        name: 'Drama',
+        clusterName: 'Vaayu',
+        type: 'cultural',
+        desc: 'Drama for first years',
+        cupName: 'dramacup',
+        r1Date: '15 june',
+        r2Date: '18 june'),
+  ];
+  Event testevent = Event(
+      name: 'Drama',
+      clusterName: 'Vaayu',
+      type: 'cultural',
+      desc: 'Drama for first years',
+      cupName: 'dramacup',
+      r1Date: '15 june',
+      r2Date: '18 june');
+
+  @override
+  void initState() {
+    super.initState();
+    _tabController = TabController(length: 5, vsync: this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        key: _scaffoldKey,
+        appBar: AppBar(
```

Comment:
We are not having an app bar in the app. so remove this.
And see how the navicon was implemented in authpage